### ISSUE TITLE
css to apply multiple times on same category?

### ISSUE THREAD

**Issue by rebelporg** [mdelobelle/obsidian_supercharged_links#94]
Hi, this plugin is incredible.

I'm trying to have _both_ the prepend _and_ the font changes/background color apply for a _single_ category. E.g., If someone is a "person" they both get an emoji prepend _and_ a blue background.

Using the style settings editor:

Doing a prepend and a background color means that the prepend is not itself colored, despite the preview in the style settings plugin. (See attached.)

Using the css editor:

Doing a prepend and a background color means that both are background colored separately, rather than together. (See attached.)

I'm wondering if this is due to my coding, but knowing that the prepends are apparently not colorable through the style settings editor means it might be something with the plugin itself. (?) Thank you!

Style settings preview:
![Screen Shot 2022-02-17 at 12 20 51 PM](https://user-images.githubusercontent.com/60893004/154536010-964694bf-2775-4e4e-a9bb-cfea3a1e6c24.png)

Style settings actual:
![Screen Shot 2022-02-17 at 12 20 57 PM](https://user-images.githubusercontent.com/60893004/154536013-f45fbe78-b932-4f7c-8a34-ade2b409e331.png)

CSS script:
![Screen Shot 2022-02-17 at 12 14 44 PM](https://user-images.githubusercontent.com/60893004/154536016-b2eb7cd2-438f-4d57-a955-7f6088f18a30.png)

CSS actual:
![Screen Shot 2022-02-17 at 12 15 05 PM](https://user-images.githubusercontent.com/60893004/154536015-d2d9d67e-6469-41d4-ad3f-945631483b4b.png)

**Comment by HEmile** [comment]
Yes, good observation. Note that this only happens in live preview. It should look like the Style settings preview in the reading mode. 
This is because, as you can see in the CSS actual screenshot, it creates these (imo) ugly round shapes around the prepend... unfortunately this is either really hard or impossible to fix, which is why i decided not to background color those in live preview

**Comment by rebelporg** [comment]
ok great! no problem, just as long as I know it's _not possible_, then I will make a different format and stop playing around with it :)

THANK YOU FOR THIS GREAT PLUGIN! 👯

**Comment by rebelporg** [comment]
Actually, I've realized that as long as you don't do a border radius, it looks pretty good even in live preview, while you're not clicked on the link itself. Just for anyone who may read this in the future:

![Screen Shot 2022-02-17 at 1 40 33 PM](https://user-images.githubusercontent.com/60893004/154
![Screen Shot 2022-02-17 at 1 40 15 PM](https://user-images.githubusercontent.com/60893004/154548977-d6c90002-ff2a-4cbd-b964-6d7af10041a3.png)
![Screen Shot 2022-02-17 at 1 40 22 PM](https://user-images.githubusercontent.com/60893004/154548978-59834c2e-3a67-4264-bb4a-cd32af9147e7.png)
548944-990d88bf-d485-4ae5-8d2f-a8daf0c0cb8a.png)

**Comment by HEmile** [comment]
Yes, that is true... but i don't think it looks as great in general
